packages/view-types/spatial/package.json

```diff
@@ -44,7 +44,9 @@
     "lodash-es": "^4.17.21",
     "math.gl": "^3.5.6",
     "mathjs": "^9.2.0",
-    "short-number": "^1.0.6"
+    "short-number": "^1.0.6",
+    "@deck.gl/core": "~8.8.6",
+    "@deck.gl/layers": "~8.8.6"
```

Comment:
these changes won't be merged in -- they are temporary to enable local testing with the changes of ScaleBarLayer